docs/tutorials/task.cis-to-component-definition/transformation.md

```diff
@@ -0,0 +1,1092 @@
+
+```
+(venv.trestle)$ mkdir -p adjunct-data/cis-benchmarks
+(venv.trestle)$ mkdir -p adjunct-data/trestle-configs
+```
+
+- Fetch ComplianceAsCode profile data.
+
+```
+(venv.trestle)$ curl 'https://raw.githubusercontent.com/ComplianceAsCode/content/master/products/ocp4/profiles/cis-node.profile' > adjunct-data/cis-benchmarks/cis-node.profile
+(venv.trestle)$ curl 'https://raw.githubusercontent.com/ComplianceAsCode/content/master/products/ocp4/profiles/cis-node.profile' > adjunct-data/cis-benchmarks/cis.profile
+
+```
+
+- Fetch trestle task files.
+
+```
+(venv.trestle)$ curl 'https://https://github.com/IBM/compliance-trestle/blob/develop/docs/tutorials/task.cis-to-component-definition/demo-cis-to-component-definition.config' > adjunct-data/task-files/demo-cis-to-component-definition.config
+(venv.trestle)$ curl 'https://https://github.com/IBM/compliance-trestle/blob/develop/docs/tutorials/task.cis-to-component-definition/selected_rules.json' > adjunct-data/task-files/selected_rules.json
+(venv.trestle)$ curl 'https://https://github.com/IBM/compliance-trestle/blob/develop/docs/tutorials/task.cis-to-component-definition/enabled_rules.json' > adjunct-data/task-files/enabled_rules.json
+```
+
+<details>
+<summary>demo-cis-to-component-definition.config</summary>
+
+```
+[task.cis-to-component-definition]
+
+output-dir = component-definitions/cis
+
+org-name = International Business Machines
+org-remarks = IBM
+
+component-name = OSCO
+
+profile-name = OCP4 CIS-benchmark v4
+profile-mnemonic = ocp4-cis-node
+profile-ns = https://github.com/ComplianceAsCode/content/tree/master/ocp4
+profile-version = 1.1
+profile-check-version = 0.1.58
+
+profile-type = OCP4
+profile-list = cis-node cis
+
+profile-file.cis-node = adjunct-data/cis-benchmarks/cis-node.profile
+profile-url.cis-node = https://github.com/ComplianceAsCode/content/blob/master/products/ocp4/profiles/cis-node.profile
+profile-title.cis-node = CIS Red Hat OpenShift Container Platform 4 Benchmark
+
+profile-file.cis = adjunct-data/cis-benchmarks/cis.profile
+profile-url.cis = https://github.com/ComplianceAsCode/content/blob/master/products/ocp4/profiles/cis.profile
+profile-title.cis = CIS Red Hat OpenShift Container Platform 4 Benchmark
+
+selected-rules  = adjunct-data/task-files/selected_rules.json
+enabled-rules  = adjunct-data/task-files/enabled_rules.json
+```
+
+</details>
+
+<details>
+<summary>selected_rules.json</summary>
+
+```
+{
+  "selected": [
+    "file_permissions_kube_apiserver",
+    "file_owner_kube_apiserver",
+    "file_groupowner_kube_apiserver",
+    "file_permissions_kube_controller_manager",
+    "file_owner_kube_controller_manager",
+    "file_groupowner_kube_controller_manager",
+    "file_permissions_scheduler",
+    "file_owner_kube_scheduler",
+    "file_groupowner_kube_scheduler",
+    "file_permissions_etcd_member"
+  ]
+}
+
+```
+
+</details>
+
+<details>
+<summary>enabled_rules.json</summary>
+```
+{
+  "enabled": [
+    "file_permissions_kube_apiserver",
```

Comment:
This is not rendered properly as markdown.  May need newline before triple back quote.

Reply:
Also if after your triple quotes (in github) you include a file type (e.g. json). I've put an example into your PR comment above.

Reply:
Fixed triple quotes. Don't understand Chris' comment yet...